homeassistant/components/portainer/quality_scale.yaml

```diff
@@ -37,14 +37,14 @@ rules:
     status: exempt
     comment: |
       No explicit parallel updates are defined.
-  reauthentication-flow:
-    status: todo
-    comment: |
-      No reauthentication flow is defined. It will be done in a next iteration.
+  reauthentication-flow: done
   test-coverage: done
   # Gold
   devices: done
-  diagnostics: todo
+  diagnostics:
+    status: exempt
+    comment: |
+      Diagnostics are implemented, the PR is still open. Waiting...
```

Comment:
You can't mark it as `exempt` because https://github.com/home-assistant/core/pull/153126 it not accepted yet, I left a minor comment, but there are comments from other reviewers there.

I suggest to update https://github.com/home-assistant/core/pull/153126 and mark it as `done` there so once it is merged the status will be updated and it won't be needed to be reviewed again.